### PR TITLE
Fix from feedback 29-11-2019

### PR DESCRIPTION
**Pull request by haik1803**

### CODE REVIEW

**Comment by jchacana** [comment]
image

Hello @haik1803 , your repo is not ignoring some IntellijJ specific files (look at the attached image)

Also, how are you providing the functionality from previous stages? For example, reading transactions from a file, providing it as a parameter, etc?

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -13,3 +13,6 @@ spring.datasource.password=Pi@123456
 
 spring.jpa.hibernate.ddl-auto=update
 spring.jpa.show-sql=true
+
+########## Initial Account Data ########
+spring.csv.path=D:\\Demo\\data.csv
```
Comment (jchacana) [inline src/main/resources/application.properties]: Careful with this. Either mention it in your README and what is it for or you can distribute it in a more standard folder (src/main/resources/data.csv)

Comment (haik1803) [reply src/main/resources/application.properties]: I have fixed these. Can you check it for me?

**Comment by jchacana** [comment]
Ok we'll merge

**Pull request merged**